docs/hoon/library/1a.md

```diff
@@ -62,33 +76,43 @@ An atom.
 Source
 ------
 
-    ++  dec                                                 ::  decrement
+    ++  dec                                               
       ~/  %dec
-      |=  a/@
-      ~|  %decrement-underflow
+      |=  a=@
+      ~_  leaf+"decrement-underflow"
       ?<  =(0 a)
       =+  b=0
       |-  ^-  @
       ?:  =(a +(b))  b
       $(b +(b))
-
+::
 Examples
 --------
 
     > (dec 7)
     6
+
     > (dec 0)
     ! decrement-underflow
     ! exit
 
+    > (dec 'b')
```

Comment:
Let's not multiply examples too much.  We should drop the first cord example, maybe the second too.  I like the hex example, but that should be enough to clue people in on using `dec` with other auras.